angular-ngrx-scss/src/app/state/auth/auth.effects.ts

```diff
@@ -49,9 +70,40 @@ export class AuthEffects {
     );
   });
 
+  /**
+   * Gets authenticated user's name, photo, and email
+   */
+  fetchAuthUserData$ = createEffect(() => {
+    return this.actions$.pipe(
+      ofType(saveUserTokenSuccess, userTokenExists),
+      switchMap(() =>
+        this.store.select(selectAuthUserName).pipe(
+          skipWhile((name) => name.length > 0),
```

Comment:
just curious. What is the intention here for using `skipWhile`?

Reply:
wouldn't it make more sense to use `distinctUntilChanged` then?

Reply:
Oops missed that one, this is fixed now.